Runtimes/Core/runtime/CMakeLists.txt

```diff
@@ -95,7 +95,8 @@ target_include_directories(swiftRuntime PRIVATE
 
 target_link_libraries(swiftRuntime PRIVATE
   $<$<PLATFORM_ID:Windows>:User32>
-  swiftShims)
+  swiftShims
+  swiftDemangling)
```

Comment:
We shouldn't need this. I don't see `SWIFT_INLINE_NAMESPACE` getting used in `swiftRuntime`.

Reply:
It is used indirectly since a few sources in `runtime` include Demangling headers, which in turn include `namespaceMacros.h` -- if I don't add that line, I get linker errors because we are not able to find symbols like `mangleNode`.

Reply:
Ah, good catch.